opm/common/data/SimulationDataContainer.hpp

```diff
@@ -86,6 +94,7 @@ namespace Opm {
         const std::vector<double>& facepressure() const;
         const std::vector<double>& faceflux    () const;
 
+        const std::unordered_map<std::string, std::vector<double>> cellData() const;
```

Comment:
Oops, I should have caught that...! That is certainly a demonstration of the value of type aliases (it makes such declarations easier to read, and such errors easier to spot).